irispy/iris_tools.py

```diff
@@ -435,7 +435,8 @@ def _gaussian1d_on_linear_bg(x, amplitude=None, mean=None, standard_deviation=No
     return amplitude * np.exp(-((x - mean) / standard_deviation) ** 2) + constant_term + linear_term * x
 
 
-def _calculate_orbital_wavelength_variation(data_array, date_data_created, slit_pixel_range=None, spline_smoothing=False, fit_individual_profiles=False, spacecraft_velocity=None, orbital_phase=None, roll_angle=None):
+def _calculate_orbital_wavelength_variation(data_array, date_data_created, \ slit_pixel_range=None, spline_smoothing=False, fit_individual_profiles=False, \
```

Comment:
the ```\```'s aren't necessary.  If there's a line break but the command is not closed properly, e.g. a bracket is missing or the line ends in a comma, Python automatically assumes it's one line of code stretched over two lines of text.

Reply:
But I thought this way it would be clearer where the line breaks are... otherwise it could look kind of messy in some places as I have just saw moments ago. Is it okay to keep them?

Reply:
Okay, I will go ahead and remove the `\`'s then...

Reply:
It's not in keeping with standard Python coding style unless they are explicitly needed.

Reply:
Done, all redundant `\`'s removed from `iris_tools.py`

Reply:
Done the same for `spectrograph.py` as well.